packages/uui-caret/lib/uui-caret.element.ts

```diff
@@ -25,8 +25,17 @@ export class UUICaretElement extends LitElement {
   }
 
   render() {
-    return html`<svg viewBox="0 0 512 512">
-      <path d="M 255.125 400.35 L 88.193 188.765 H 422.055 Z"></path>
+    return html`<svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
```

Comment:
Do we need both width/height and viewBox?

Reply:
You right, no difference in two of the cases (Expand and Caret)

But it does make a difference for the sorter component — I do though think it can be removed there as well, but it is not as simple so that would require some re-write of the styling.